docs/github-app.adoc

```diff
@@ -52,6 +52,13 @@ Permissions this plugin uses:
 - Pull requests: Read-only
 - Webhooks (optional) - If you want the plugin to manage webhooks for you, Read and Write
 
+You are not required to use webhooks but it strongly recommeded over alternatives such as polling. To use webhooks, you can give the your GitHub App "Read and Write" permission to Webhooks (noted above) and the plugin will configure your repositories to send the appropriate events.  Otherwise you can manually configure the app handle all events needed by this plugin, in which case the app will not need webhook permissions.
```

Comment:
```suggestion
You are not required to use webhooks but it strongly recommeded over alternatives such as polling. To use webhooks, you can give your GitHub App "Read and Write" permission to Webhooks (noted above) and the plugin will configure your repositories to send the appropriate events.  Otherwise you can manually configure the app handle all events needed by this plugin, in which case the app will not need webhook permissions.
```